README.md

```diff
@@ -1,56 +1,60 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Williams Forex App
 
-# Project Name
+> This is a Web APP Focused only on currency trading, covers some of the major currency pairs traded worldwide, such as EUR/USD, USD/JPY and GBP/USD. The users are able to view live prices for the major Forex pairs, and to learn what factors that impact their price movements.
 
-> Description the project.
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
+> Presentation Video [Link](https://www.loom.com/share/5699c3002f464c01a21b434e73408307)
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Visual Studio Code, Git, GitHub using Gitflow.
+- React components.
+- Redux toolkit.
+- Modular CSS.
+- Jest.
 
-## Live Demo (if available)
+## Quick view
+> ![](./screenshot-fx-app.png)
+> ![](./screenshot-fx-app-info.png)
 
-[Live Demo Link](https://livedemo.com)
 
+## Live Demo
 
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+[Press here to see this project live](https://forexappwilliamscch.netlify.app/)
 
+## Getting Started
 
-To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
-
-### Setup
+- To get a local copy up and running, copy the SSH key of the repository and run `git clone` in your terminal.
 
 ### Install
+- `npm install`: Install all the dependencies required locally.
 
-### Usage
+### Setup
+- `npm run build`: Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
 
-### Run tests
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
 
 ### Deployment
+- `npm start`: Runs the app in the development mode.
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
 
+### Run tests 
+- `npm test`: Launches the test runner in the interactive watch mode.\
 
-## Authors
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+## 👤 Author
 
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **Williams Colmenares**
+- GitHub: [@williamscch](https://github.com/williamscch)
+- Twitter: [@wdavidcch](https://twitter.com/wdavidcch)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/williamscolmenaresch/)
 
 ## 🤝 Contributing
```

Comment:
The link to your issues page is broken because you did not include the URL for your issues page. Kindly add a working link to your issues page to make your README more professional.

Reply:
Okay